### DISCUSSION TITLE
synology snmp-exporter

### DISCUSSION THREAD

**Discussion by dstewen** [h3mmy/bloopySphere#6511] [Q&A] [answered]
You mention in the README that snmp-exporter-synology has a default config that doesn't fit in a configMap, but it appears you are building a configMap from it. How?

Is your snmp.yml customized so that it fits into the configMap?

I'm trying to get a similar setup working with a different configMap and the pod deploys but goes into a crashloopbackoff because of the configMap.

**Comment by h3mmy** [comment, marked as answer]
I should probably organize my notes better. 😅

The nested README in the [synology-nas](https://github.com/h3mmy/bloopySphere/tree/404cdc4d023f6497716b681b6922ec8bab7d28a9/cluster/apps/monitoring/snmp-exporter/synology-nas) folder has a note that I used the generator from [prometheus/snmp_exporter](https://github.com/prometheus/snmp_exporter/tree/main/generator) to make the "customized" `snmp.yaml`

My [generator.yaml](https://github.com/h3mmy/bloopySphere/blob/404cdc4d023f6497716b681b6922ec8bab7d28a9/cluster/apps/monitoring/snmp-exporter/synology-nas/generator.yml) has comments explaining where I grabbed the MiBs from and what some of the options are used for.
